server/routes/additionalDaysAwardedRoutes.ts

```diff
@@ -1,33 +1,36 @@
 import { RequestHandler } from 'express'
 import PrisonerService from '../services/prisonerService'
-import { AdaIntercept, AdasToReview, PadasToReview } from '../@types/AdaTypes'
+import { AdasToReview, PadasToReview } from '../@types/AdaTypes'
 import { Message } from '../model/adjustmentsHubViewModel'
 import PadaForm from '../model/padaForm'
 import AdditionalDaysAwardedBackendService from '../services/additionalDaysAwardedBackendService'
+import AdjustmentsService from '../services/adjustmentsService'
 
 export default class AdditionalDaysAwardedRoutes {
   constructor(
     private readonly prisonerService: PrisonerService,
     private readonly additionalDaysAwardedBackendService: AdditionalDaysAwardedBackendService,
+    private readonly adjustmentsService: AdjustmentsService,
   ) {}
 
   public intercept: RequestHandler = async (req, res): Promise<void> => {
     const { token, activeCaseLoadId } = res.locals.user
     const { nomsId } = req.params
     const { prisonerNumber } = res.locals.prisoner
-    const intercept: AdaIntercept = await this.additionalDaysAwardedBackendService.shouldIntercept(
-      prisonerNumber,
-      token,
-      activeCaseLoadId,
-    )
+    const response = await this.adjustmentsService.getAdaAdjudicationDetails(prisonerNumber, token, activeCaseLoadId)
 
-    if (intercept.type === 'NONE') {
+    if (response.intercept.type === 'NONE') {
       return res.redirect(`/${nomsId}`)
     }
 
+    let numOfAdaAdjudications = 0
```

Comment:
1. So i think this variable should be re-named. Its the number of adjudications which could be created as an adjustment, which is why we're only interesting in pending approval and prospective adjudications.
2.  Also we don't need to know the number, we only care if its non zero, so might be better to pass the value to the model as a boolean.
3. This logic will probably be re-used in all the other screens, so might make sense to move it to the backend.